### PR TITLE
Fix ansible deploy

### PR DESCRIPTION
**Pull request by leobalter**
Now it points to the right repo and sets ansible to send the ssh agent
forward, without requiring to link an id anymore.

cc @jugglinmike

### CODE REVIEW

**Comment by tkellen** [comment]
Ansible looks for it's configuration file in the following locations (in this order):
- ANSIBLE_CONFIG (an environment variable)
- ansible.cfg (in the current directory)
- .ansible.cfg (in the home directory)
- /etc/ansible/ansible.cfg

The current configuration won't work unless you alter the package.json scripts for deploy and production to cd into the ansible folder first.  Also, they don't need `--ask-sudo-pass` after them any longer. You could also do as @jugglinmike did and completely remove the npm script commands and update the readme to include instructions for running the commands "manually".  I'm not wild about losing the npm script shorthand, but that's just my opinion!

**Comment by leobalter** [comment]
@tkellen 

I tried some different scenarios to understand everything right, now I moved the ansible.cfg to the project's root, so we can keep the npm shorthand.

**Comment by tkellen** [comment]
Hrm.  Are we sure we need that ansible config?  I'm doing git-based deployment on the bocoup api without it.

File: ansible.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[defaults]
+ask_sudo_pass = True
+transport = ssh
+
+[ssh_connection]
+ssh_args = -o ForwardAgent=yes
```
Comment (leobalter) [inline ansible.cfg]: @tkellen the config file is being used here because of the Forward Agent option. So we can drop the `
deploy_key_path` variable.

Comment (leobalter) [inline ansible.cfg]: but it's working without the config file if I use the --ask-sudo-pass parameter again.

**Comment by leobalter** [comment]
By bringing back `--ask-sudo-pass` seems better than setting the cfg file as we use the npm shorthand.

@tkellen: does it look better now?

**Comment by tkellen** [comment]
Looking good! I'm pretty sure you don't need the deploy key path either--ssh agent forwarding is handled here:
https://github.com/bocoup/jqfundamentals.com/blob/master/deploy/ansible/roles/configure/tasks/main.yml#L21-L28

**Comment by leobalter** [comment]
We need to merge this up, we've got fix updates on the website and a deploy is imminent. 

I'll check this up.

**Comment by leobalter** [comment]
@tkellen let's merge this one too?

**Pull request merged**